Back-To-School/Assets/BackToSchool/Scripts/Enemies/Enemy.cs

```diff
@@ -30,55 +31,46 @@ public class Enemy : MonoBehaviour
         private bool _isBusy;
         private bool _isDead;
 
-        public void GetDamage()
+        public void TakeDamage(int damage)
         {
-            _currentHealth--;
-            if (OnHealthChanged != null) OnHealthChanged(this, new EnemyArgs((float) _currentHealth / _maxHealth));
+            _currentHealth -= damage;
+            HealthChanged?.Invoke(_currentHealth, _maxHealth, damage);
```

Comment:
okay may be 'damage' parameter is unnecessary, because if we need delta between old value and new value, we can calculate it in UI just doing subtraction. But maxHP is required, because UI shows HP in %